website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -214,6 +214,32 @@ exports.handler = async function (request, response) {
 };
 ```
 
+### Decoding the token
+
+You can decode the JWT token with [JWT.io](https://jwt.io/). In the decoded version, you will see the following fields.
+
+```json
+{
+  "exp": 1600092401,
+  "sub": "<subject>",
+  "iss": "<hostname>",
+  "aud": "<audience-url>",
+  "type": "exchange",
+  "<hostname>/claims/project_key": "<project-key>",
+  "iat": 1600092341
+}
+```
+
+These fields are according to specification **Register Claim Names** and consist of the following:
+
+- `exp`: This claim identifies the **expiration time**. The claims requires to be before the current time in order to accepted for processing.
+- `sub`: This claim identifies the **subject**. The value of `sub` issued by the API Gateway for Custom Applications is the user-ID for Merchant Center.
```

Comment:
Please use the term `Merchant Center API Gateway` instead of `API Gateway`.

Maybe we can just say it's the "ID of the commercetools platform user".